frontend/src/features/admin/constants/index.tsx

```diff
@@ -35,3 +35,12 @@ export const columnDefinitions: ColumnWithProps<IAgencyRecord>[] = [
     },
   },
 ];
+
+export const adminAreasColumnDefinistions: ColumnWithProps<any>[] = [
```

Comment:
Don't place functions in an `index.ts` file.  Move them to a `utils.ts` file.  Index files should only be used to enable easy referencing of other files in the folder.

Reply:
Woops yes this was a mistake